### PR TITLE
Fix/in place edit quirks

### PR DESCRIPTION
**Pull request by ulferts**
Take a perfectly working solution and make it work in IE...

https://community.openproject.org/work_packages/17057
https://community.openproject.org/work_packages/19351

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://ulferts-fix-in-place-edit-quirks.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://ulferts-fix-in-place-edit-quirks.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by ulferts** [comment]
https://community.openproject.org/work_packages/19351

**Comment by NobodysNightmare** [comment]
Funtional testing revealed, that you fixed all the bugs that you promised:
- one that is IE only
- one that is FF only
- one that is chrome only
- two that are cross browser (though I did not check whether the X is now really really centered)

I am still wondering what let you write this micro-rant on IE...

File: frontend/app/ui_components/wiki-toolbar-directive.js [hunk 1]
```diff
@@ -27,7 +27,7 @@
 //++
 
 module.exports = function() {
-  var HELP_LINK_HTML = '<a href="/help/wiki_syntax" title="' + I18n.t('js.inplace.link_formatting_help') + '" class="icon icon-help" onclick="window.open(&quot;/help/wiki_syntax&quot;, &quot;&quot;, &quot;resizable=yes, location=no, width=600, height=640, menubar=no, status=no, scrollbars=yes&quot;); return false;"><span class="hidden-for-sighted">' + I18n.t('js.inplace.link_formatting_help') + '</span></a>',
```
Comment (NobodysNightmare) [inline frontend/app/ui_components/wiki-toolbar-directive.js:30]: I was irritated for a short moment because you removed the `hidden-for-sighted` span without replacement.

I assume that is suffices for a screenreader to have the title set correctly. Though I am wondering why the span existed in the first place.

**tl;dr:** You did not break accessibility, did you?

**Comment by floriank** [comment]
Looks good to me so far - the comment by @NobodysNightmare should be addressed though

**Comment by NobodysNightmare** [comment]
Waiting for reaction on my accessibility comment. Otherwise :+1:

File: frontend/app/ui_components/wiki-toolbar-directive.js (hunk 1 above)
Comment (ulferts) [inline frontend/app/ui_components/wiki-toolbar-directive.js:30]: I assumed the title to suffice, but apparently I was wrong http://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/

Thanks for spotting.
